Application/EdFi.Ods.Tests/EdFi.Ods.Security/AuthorizationStrategies/Relationships/AuthorizationSegmentsToFilterConverterTests.cs

```diff
@@ -264,11 +264,6 @@ public void Should_return_filters_for_each_associated_EdOrg_type()
                 _actualFilters[0].ClaimEndpointName.ShouldBe("LocalEducationAgencyId");
```

Comment:
I'm not going to review these tests until the implementation stabilizes. I'll revisit these when the rest of the implementation is stable.

Reply:
Please review if you think implementation is stable now .